apps/user-portal/src/components/account-recovery-security-qs.tsx

```diff
@@ -248,54 +250,72 @@ class SecurityQuestionsComponentInner extends React.Component<ComponentProps, an
 
     public render() {
         const { t } = this.props;
-        const {challenges} = this.state;
+        const { challenges } = this.state;
         const listItems = () => {
             if (challenges.answers && (challenges.answers.length > 0) && (!this.state.isEdit)) {
                 return (
-                    <List divided verticalAlign="middle" className="main-content-inner">
-                        {
-                            challenges.answers.map((answer) => {
-                                return (
-                                    <List.Item className="inner-list-item">
-                                        <Grid padded>
-                                            <Grid.Row columns={ 2 }>
-                                                <Grid.Column width={ 16 } className="first-column">
-                                                    <List.Content>{ answer.question }</List.Content>
-                                                </Grid.Column>
-                                            </Grid.Row>
-                                        </Grid>
-                                    </List.Item>
-                                );
-                            })
-                        }
-                    </List>
+                    <Grid padded>
+                        <Grid.Row columns={2}>
+                            <Grid.Column width={11} className="first-column">
+                                <List.Content floated="left">
+                                    <ThemeIcon
+                                        icon={MFAIcons.sms}
```

Comment:
shall we have a separate icon for security questions too?

Reply:
A new icon has been added for security questions. 
![image](https://user-images.githubusercontent.com/20745428/66900298-2b6eac00-f01a-11e9-9342-f06e441ba159.png)